jobs/buildhub/inventory_to_records.py

```diff
@@ -65,6 +74,16 @@
                 headers=headers,
                 timeout=None
             ) as response:
+                # Calling 'response.raise_for_status()' when the response
+                # status code is 404 will trigger a ClientResponseError
+                # which is part of the backoff (see decorator).
+                # *Sometimes* a 404 response is actually something that can
+                # happen due to the latency of a slow update in
+                # archive.mozilla.org so if you just back off and try again
+                # in a couple of minutes it will work.
```

Comment:
This comment is redacted in a way that is not directly related to the condition just below. Maybe it should mention that raising JSONFileNotFound will exit without retry or something...

Reply:
I think I'll move it all to the doc string of the function instead.